website/docs/tutorials/07-testing/02-cheatcodes/expect-call.md

```diff
@@ -34,17 +36,30 @@ func test_expect_call_success{syscall_ptr: felt*, range_check_ptr}() {
   tempvar ctr_addr_b;
   %{ ids.ctr_addr_b = context.ctr_addr_b %}
 
-  %{
-    expect_call(ids.ctr_addr_a, "increase_balance", [5, 6, 7])
-    expect_call(ids.ctr_addr_b, "increase_balance", [1, 2, 3])
-  %}
+  %{ stop_a = expect_call(ids.ctr_addr_a, "increase_balance", [5, 6, 7]) %}
```

Comment:
```suggestion
  %{ assert_increase_balance_567_called = expect_call(ids.ctr_addr_a, "increase_balance", [5, 6, 7]) %}
```
I think we should use meaningful names in the example.

Reply:
[update docs](https://github.com/software-mansion/protostar/pull/1213/commits/21bec7875f678068bf55e1b03705fe0cb2995d05)